### ISSUE TITLE
Automatically updating children page's slug to reflect parent title change

### ISSUE THREAD

**Issue by jianghong** [stephenmcd/mezzanine#831]
Hi everyone, I have an issue that I asked about on the mezzanine google group:

https://groups.google.com/forum/#!topic/mezzanine-users/9FRTeTut3oo

Steps to recreate the issue:
1. Create new page at the top level (this is the parent page).
2. Create a child page to that parent page.
3. Check that both links are working (they are).
4. Change title of the parent page, delete the url to leave it blank so it autogenerates to match the new title.
5. The child page url is not updated and thus is a broken link now.

Basically I was wondering if there's a solution for updating the slugs of all of a page's children if that page's title and slug are updated. Currently it does not.
